DuckDuckGo/BookmarksDataSource.swift

```diff
@@ -74,4 +77,9 @@ class BookmarksDataSource: NSObject, UITableViewDataSource {
     func tableView(_ tableView: UITableView, moveRowAt sourceIndexPath: IndexPath, to destinationIndexPath: IndexPath) {
         bookmarksManager.move(itemAtIndex: sourceIndexPath.row, to: destinationIndexPath.row)
     }
+    
+    private func updateColor(_ cell: UITableViewCell) {
+        if #available(iOS 10, *) { return }
```

Comment:
We used to have the color in IB but it's easy to see it there and delete it thinking it's not needed. I like this better, it makes the reason apparent.